vello/src/shaders.rs

```diff
@@ -62,7 +62,7 @@ pub fn full_shaders(
     #[cfg(feature = "hot_reload")]
     let mut shaders = vello_shaders::compile::ShaderInfo::from_dir(concat!(
```

Comment:
Can `vello_shaders` instead export the directory the shaders are located in?

Reply:
You mean the `shader` directory name, or some more absolute path?

Reply:
Alright I added a new function that returns an absolute path to the shader directory. It depends on using Cargo and the path is computed at compile time.

If someone has a build cache and then moves the directories around, they will need to `cargo clean` or trigger a rebuild in another way.